docs/operational_guide/placement_changes.md

```diff
@@ -0,0 +1,132 @@
+# Placement Changes
+
+## Overview
+
+M3DB was designed from the ground up to a be a distributed / clustered database that is isolation group aware. Clusters will seamlessly scale with your data, and you can start with a cluster as small as 3 nodes and grow it to a size of several hundred nodes with no downtime or expensive migrations.
```

Comment:
I get that the `3 nodes` comment is referring to clusters, but I wonder if this will confuse people in thinking it's not possible to test M3DB on their own (single) machine?

Reply:
Yeah, perhaps just say "small number of nodes".